packages/plugins/ganache/src/index.js

```diff
@@ -2,7 +2,9 @@ class Ganache {
   constructor(embark) {
     embark.events.request('blockchain:vm:register', () => {
       const ganache = require('ganache-cli');
-      return ganache.provider();
+      // Default to 8000000, which is the server default
+      // Somehow, the provider default is 6721975
```

Comment:
`6721975` is the specified default gas limit in ganache. We're also using it in other places to ensure complex smart contracts can be deployed. See https://github.com/embarklabs/embark/pull/1650 and https://github.com/embarklabs/embark/pull/1687 for context.

Reply:
I checked and those values are not used anymore